packages/experimental/Tabs/src/Tabs.styling.macos.ts

```diff
@@ -0,0 +1,41 @@
+import { tabsName, TabsTokens, TabsSlotProps, TabsProps } from './Tabs.types';
+import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { fontStyles } from '@fluentui-react-native/tokens';
+
+import { defaultTabsTokens } from './TabsTokens';
+
+export const stylingSettings: UseStylingOptions<TabsProps, TabsSlotProps, TabsTokens> = {
+  tokens: [defaultTabsTokens, tabsName],
+  slotProps: {
+    root: buildProps(
+      () => ({
+        style: {
+          display: 'flex',
+          minHeight: 32,
+          minWidth: 80,
+          marginLeft: 4,
```

Comment:
The only style that differs between macOS, win32, and windows for Tabs are margins (margin, marginTop, marginStart, etc.) which are placed on different slots for each platform. I'd like to reduce the amount of platform specific files we have  for this control but I didn't see how else to apply these styles w/o creating a separate file for each platform. If there's a better way to do this, let me know.

Reply:
Is that just because windows doesn't have focusZone? I think that was the main reason slots were different for windows. Not sure about win32

Reply:
No, (unless I'm misunderstanding you) I don't think this has to do with focusZone at all. It's just that the different platforms have different styles for the Tabs controls w.r.t. margins on the root, label, and stack.

Reply:
Like macOS places margins on the root and the stack while windows places margins on the label and the stack and win32 doesn't set margins at all. We don't even reference the container slot (which is FocusZone) in this file.

Reply:
Do they have the same slots, but different margins on each slot?
That seems weird, can't imagine the style would change that much....

I thought it was that windows vs win32 vs macOS had different slots altogether, my bad.

Reply:
@ksiler I'd consider making a `getTabsMargins()` function which returns different values for each platform (so that file would be forked), and then use it to build the style using the spread operator. Then you could consolidate the rest of this file into one file.

Reply:
Yeah, they all reference the same slots in the styling file but margins are placed on different slots. I just copied the styles over directly from Tabs on the old framework. There aren't official designs for Tabs on macOS or Windows so some of these styles are best guesses and may change in the future so maybe that's why it seems weird?

Reply:
@rurikoaraki Thanks for the suggestion! But I'm a bit confused on how that would work if I need to supply different margins to different slots. For example, macOS uses `marginLeft: 4` on the root slot and `marginTop: 6` on the stack slot. If I were to use `...getTabsMargins()` under each slot, how would I know which margin style to return?

Reply:
@ksiler Oh I didn't realize multiple slots had different margins. You could make a function for each one (i.e. `getRootMargins()`, `getStackMargins()`) and define each as needed in a platform-forked file. So like, a `TabsMargins.ts` has the default values for `getRootMargins()` and `getStackMargins()` but `TabsMargins.windows.ts` has whatever you need for UWP specifically.

Reply:
@ksiler I think that's up to you. The one downside of leaving it as is is that it's not clear which styles are common across platforms, so if someone tweaks one style, that wouldn't propagate to other components automatically and it's easy to miss that other platforms may need changing, too. But you're right that we might end up having more separate styling for each platform, so it's valid to leave it as is if you foresee that the styling will diverge a decent amount on other platforms.

Reply:
@rurikoaraki  I see, thanks for your perspective!